### ISSUE TITLE
T5 learning rate

### ISSUE THREAD

**Issue by Ssunbell** [BoocampMonster/korean-relation-extraction#32]
- 0.0001(1e-4)에서 val loss가 가장 best

<img width="1002" alt="스크린샷 2022-11-29 오후 6 00 07" src="https://user-images.githubusercontent.com/97590480/204484717-096ea05e-3098-4703-801a-489e7dd33502.png">

- 0.0002(2e-4)가 그다음
<img width="1001" alt="스크린샷 2022-11-29 오후 7 49 02" src="https://user-images.githubusercontent.com/97590480/204509708-f0024760-6b93-4419-ac13-085f2408aa07.png">

### 1e-4가 val_loss는 더 낮지만 2e-4가 f1이 더 높으므로 둘중에 하나를 선택하는 것이 좋아보임
@hundredeuk2
